boards/arm/beagle_bcf/beagleconnect_freedom.dts

```diff
@@ -127,16 +118,25 @@
 		reg = <0x4>;
 	};
 
-	light: opt3001-light@44 {
+	sens_i2c: sensor-switch {
 		status = "okay";
-		compatible = "ti,opt3001";
-		reg = <0x44>;
-	};
+		compatible = "gpio-i2c-switch";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		controller = <&i2c0>;
+		gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
 
-	humidity: hdc2010-humidity@41 {
-		status = "okay";
-		compatible = "ti,hdc2010";
-		reg = <0x41>;
+		light: opt3001-light@44 {
+			status = "okay";
+			compatible = "ti,opt3001";
+			reg = <0x44>;
+		};
+
+		humidity: hdc2010-humidity@41 {
+			status = "okay";
+			compatible = "ti,hdc2010";
+			reg = <0x41>;
+		};
```

Comment:
I don't know if it's just GH, but the indentation for this looks wrong.

Reply:
It seems consistent to me on neovim and github on desktop website.